websrc/default_hands/src/default_hands_main.tsx

```diff
@@ -1070,14 +1083,7 @@ class DefaultHands extends React.Component< {}, DefaultHandsState >
 					} volume={ emptyVolume() } ref={ this.gadgetRegistryRef } />
 				</AvOrigin>
 
-				<AvOrigin path="/user/hand/left">
-					{ this.state.displayIntro && this.controllerVolumes != null && 
-						<AvHeadFacingTransform>
-							<AvTransform uniformScale = {0.07} rotateX = {30}>
-								<AvModel uri = {g_builtinModelMenuIntro}/>
-							</AvTransform>
-						</AvHeadFacingTransform>
-					}
```

Comment:
Probably worth adding this back in while you're figuring out weighted transforms.